app/controllers/participate/login.js

```diff
@@ -25,6 +26,7 @@ export default Ember.Controller.extend({
         .catch((e) => {
           if (e.status === 404) {
             this.send('toggleInvalidAuth');
+            this.set('authenticating', false);
```

Comment:
Oddly, I also can't find where `authenticating` ever gets set to `true`... am I going blind today?